views/layouts/default.lavender

```diff
@@ -52,33 +52,57 @@ body
           i.fa.fa-bars.fa-2x
       .navbar-collapse.collapse
         ul.nav.navbar-nav
-          li(class='#{active_class_index}')
-            a(href='/index.html', title='Home') Home
-          li
-            a(
-              href='https://github.com/golavender/lavender', 
-              title='GitHub', target='_blank'
-              ) GitHub
-          li
-            a(href="http://daftlabs.com", title="daft labs", target="_blank")
-              | Creators
-          li(class='#{active_class_guide} dropdown')
-            a.dropdown-toggle(href='#', data-toggle='dropdown')
-              | Guide
-              span.caret
-            ul.dropdown-menu(role='menu')
-              li
-                a(href='/guide.html') Guide Home
-              li.dropdown-header Installation
-              li
-                a(href='/guide.html#composer', title='GitHub') via Composer
-              li
-                a(href='/guide.html#github') via GitHub
-              li
-                a(href='/guide.html#usage') Usage
-              li.divider
-              li
-                a(href='/guide.html#langref') Language Reference
-          li(class='#{active_class_about}')
-            a(href='/about.html') About
+          -titles = [
+            'Home', 
+            'Github',
+            'Creators', 
+            'Guide', 
+            'Download'
+            'About'
+          ]
+          -url_page = [
+            '/index.html', 
+            'https://github.com/golavender/lavender',
+            'http://daftlabs.com',
+            '/guide.html',
+            '/download.html',
+            '/about.html'
+          ]
+          -target = [
+            '',
+            '_blank',
+            '_blank',
+            '',
+            '',
+            ''
+          ]
```

Comment:
as for why we would use objects over multiple arrays:  multiple arrays can easily get out of sorts, whereas objects are self contained and eliminates the need for keeping track of your loop index